### PR TITLE
Pipe separated format

### PR DESCRIPTION
**Pull request by meffie**
Convert the robot files to the pipe separated format style from the original spaces separated style. 

The spaces separated style has been found to be error prone and generally unfun.

The pipe separated format style has been supported since Robot Framework 2.7 so should not effect anyone running a very very old version of RF.

### CODE REVIEW

File: tests/workload/find.robot [hunk 1]
```diff
@@ -1,83 +1,84 @@
...
-    ...  |
-    Create Volume   ${VOLUME}    server=${SERVER}  part=${PARTITION}  path=${TESTPATH}
-    Create Volume   ${VOLUME}.0  server=${SERVER}  part=${PARTITION}  path=${TESTPATH}/v0
-    Create Volume   ${VOLUME}.1  server=${SERVER}  part=${PARTITION}  path=${TESTPATH}/v1
-    Create Volume   ${VOLUME}.2  server=${SERVER}  part=${PARTITION}  path=${TESTPATH}/v2
-    Create Volume   ${VOLUME}.3  server=${SERVER}  part=${PARTITION}  path=${TESTPATH}/v0/v3
-    Create File     ${TESTPATH}/v0/v3/file1  hello world\n
-    Create File     ${TESTPATH}/v2/file2     greetings\n
+|  | [Documentation] | Create a simple file tree hierarchy.
+|  | ...             | :
```
[the remark this answers is not in the document]
Comment (meffie) [reply tests/workload/find.robot:36]: Well, we'd have to look at change that replaced the | with : chars.

File: tools/robotab.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    # Trim leading and trailing whitespace from each element.
+    for i, row in enumerate(table):
+        for j, element in enumerate(row):
+            table[i][j] = element.strip()
+
+    # Find the max element width for each column.
+    for row in table:
+        for j, element in enumerate(row):
+            cw[j] = max(cw.get(j, 0), len(element))
```
[the remark this answers is not in the document]
Comment (meffie) [reply tools/robotab.py:53]: Sorry, not a fan of that. It is meant to be a table, so it's better to be column aligned imho.

File: tests/workload/find.robot (hunk 1 above)
[the remark this answers is not in the document]
Comment (meffie) [reply tests/workload/find.robot:36]: Oh, I see what happened!   This change actually preserves the existing formatting, which has four spaces after the leading pipe char, so the text is in 3rd column.  The ascii art was a bad idea anyway.

File: tools/robotab.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (CheyenneWills) [reply tools/robotab.py:53]: Not a problem.. was just a suggestion

**Comment by meffie** [comment]
On Mon, 07 Feb 2022 11:05:15 -0800
Cheyenne Wills ***@***.***> wrote:
> Not a problem.. was just a suggestion

Another point is this script is meant to be temporary, until the
upstream robotidy pipe separators are supported. If we start making
customizations like that, we will not be able to use the standard tool.


-- 
Michael Meffie ***@***.***>

**Comment by CheyenneWills** [comment]
Looks fine

**Pull request merged**